tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace BringYourOwnIdeas\Maintenance\Tests\Tasks;
+
+use BringYourOwnIdeas\Maintenance\Util\ComposerLoader;
+use SapphireTest;
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+class UpdatePackageInfoTest extends SapphireTest
+{
+    public function testGetPackageInfo()
+    {
+        $loader = new ComposerLoader(__DIR__);
+        $processor = new UpdatePackageInfo;
+        $output = $processor->getPackageInfo($loader->getLock()->packages);
```

Comment:
I chose this approach as to avoid a swing of service lookups, resulting in more of an integration test than a unit test. In this manner if someone were to wish to replace the composer loader with some kind of test stub, that would be easily accomplished - `ComposerLoader` is not the unit under test here.

Reply:
>ComposerLoader is not the unit under test here.
Exactly - it should be there as a mocked dependency of UpdatePackageInfo if it needs to be (`$this->getMock(ComposerLoader::class)`)